src/peft/tuners/ia3/config.py

```diff
@@ -28,9 +28,10 @@ class IA3Config(PeftConfig):
     Args:
         target_modules (`Union[List[str],str]`):
             The names of the modules to apply (IA)³ to. If this is specified, only the modules with the specified names
-            will be replaced. If this is not specified, modules will be chosen according to the model architecture. If
-            the architecture is not known, an error will be raised -- in this case, you should specify the target
-            modules manually.
+            will be replaced. If this is specified as 'all-linear', then all linear/Conv1D modules are chosen,
```

Comment:
I just noticed that the same update is required also for: LoHa, LoKr, OFT. Sorry for missing it so far.

Reply:
I was saving this for a future PR since I haven't actually looked into LoHa, LoKr, etc,, but have made these changes as well now!